DLA_finder/data_model/Data.py

```diff
@@ -0,0 +1,29 @@
+""" Parent class for a set of Data, training or real"""
```

Comment:
Why do we have a completely new folder named `DLA_finder` located
outside of `dla_cnn` ?